src/events/types.ts

```diff
@@ -30,6 +37,10 @@ export type BaseEventData = {
   version: string; // semver (e.g. 1.2.33)
 };
 
+export type ApprovedItem = ApprovedItemModel & {
+  authors?: ApprovedItemAuthor[];
+};
```

Comment:
what do you think about using this definition from `./src/database/types.ts` instead of declaring again here?

Reply:
Good point, thank you @jpetto! I reused the authors from there but somehow overlooked the approved item of the right shape was also already there.